zerver/openapi/zulip.yaml

```diff
@@ -3463,6 +3463,7 @@ paths:
                                         in this organization.  Null if no default has been set.
                                     message_content_delete_limit_seconds:
                                       type: integer
+                                      nullable: true
                                       description: |
                                         Messages sent more than this many seconds ago cannot be deleted
                                         with this organization's
```

Comment:
We need to document the API change here.  Every API change is double-entered, in the API changelog, and with `**Changes**` entries here.